File: JoyReactor.Accordion.WebAPI/Models/Responses/StatisticsResponse.cs
namespace JoyReactor.Accordion.WebAPI.Models.Responses;

public record StatisticsResponse
{
    public int Vectors { get; set; }

    public int ParsedTags { get; set; }
    public int EmptyTags { get; set; }
    public int ParsedPosts { get; set; }

    public int ParsedPostAttributePictures { get; set; }
    public int ParsedPostAttributePicturesWithoutVector { get; set; }
    public int ParsedPostAttributePicturesWithVector { get; set; }

    public int ParsedPostAttributeEmbeds { get; set; }

    public int ParsedBandCamps { get; set; }
    public int ParsedCoubs { get; set; }
    public int ParsedSoundClouds { get; set; }
    public int ParsedVimeos { get; set; }
    public int ParsedYoutubes { get; set; }
}

File: JoyReactor.Accordion.Web/src/services/statistics-service/statistics-response.ts
export interface StatisticsResponse {
  vectors: number,

  parsedTags: number,
  emptyTags: number,

  parsedPosts: number,

  parsedPostAttributePictures: number,
  parsedPostAttributePicturesNoContent: number,
  parsedPostAttributePicturesNoContentDueToDns: number,
  parsedPostAttributePicturesUnsupported: number,
  parsedPostAttributePicturesWithVector: number,
  parsedPostAttributePicturesWithoutVector: number,
  parsedPostAttributePicturesCheckedForDuplicates: number,

  parsedPostAttributeEmbeds: number,

  parsedBandCamps: number,
  parsedCoubs: number,
  parsedSoundClouds: number,
  parsedVimeos: number,
  parsedYouTubes: number,
}